swim/member.go

```diff
@@ -166,6 +187,23 @@ func (c Change) validateOutgoing() Change {
 	return c
 }
 
+func (c *Change) populateSubject(m *Member) {
+	if m == nil {
+		return
+	}
+	c.Address = m.Address
+	c.Incarnation = m.Incarnation
+	c.Status = m.Status
+}
+
+func (c *Change) populateSource(m *Member) {
+	if m == nil {
+		return
+	}
+	c.Source = m.Address
+	c.SourceIncarnation = m.Incarnation
+}
+
 // suspect interface
```

Comment:
the comment is hard to understand, but the function here is needed by the state transitions component which defined/defines a suspect interface from the time it was called suspect timer. Don't think I want to change it as part of this PR but we can open a new PR to update the comments on these functions to be more explaining.